resources/assets/coffee/_classes/account-edit-blocklist.coffee

```diff
@@ -0,0 +1,46 @@
+###
+#    Copyright 2015-2018 ppy Pty. Ltd.
+#
+#    This file is part of osu!web. osu!web is distributed with the hope of
+#    attracting more community contributions to the core ecosystem of osu!.
+#
+#    osu!web is free software: you can redistribute it and/or modify
+#    it under the terms of the Affero GNU General Public License version 3
+#    as published by the Free Software Foundation.
+#
+#    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+#    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#    See the GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+
+class @AccountEditBlocklist
+  bn: 'user-list__content'
+  jsClass: '.js-account-edit-blocklist'
+
+  constructor: ->
+    $(document).on 'click', @jsClass, @toggle
+    $.subscribe 'user:update', @updateBlockCount
+    @visible = false
```

Comment:
may be wrong when visiting the page again through a link instead of browser back/forward button.

1. Open page
2. toggle (show &rarr; hide)
3. open page again through header (avatar &rarr; settings)
4. toggle again (show &rarr; show)